docs/pages/jpsreport/jpsreport_method_D.md

```diff
@@ -0,0 +1,31 @@
+---
+title:  Method D
+keywords: analysis
+tags: [jpsreport, measurement]
+sidebar: jupedsim_sidebar
+folder: jpsreport
+summary: this method calculates the density based on Voronoi diagrams, which are a special kind of decomposition of a metric space determined by distances to a specified discrete set of objects in the space. 
+permalink: jpsreport_method_D.html
+last_updated: Dec 21, 2019
+---
+
+At any time the positions of the pedestrians
+can be represented as a set of points, from which the Voronoi
+diagram can be generated. 
+
+The Voronoi cell area,  $$A_i$$, for each person  $$i$$ can be obtained. 
+
```

Comment:
Why the image was removed from the detailed description of method D?

Reply:
Sorry, I forgot about it. 🙈 

Added now.